src/components/App/index.js

```diff
@@ -1,10 +1,14 @@
-import React from 'react';
+import React, {Component} from 'react';
 import logo from '../../logo.svg';
+import {withRouter} from 'react-router-dom';
```

Comment:
I don't think you use this right? Just get rid of it if you aren't using it. Also might want to pull from master cause there are some merge conflicts that will come up.